pytorch_lightning/core/hooks.py

```diff
@@ -372,6 +372,12 @@ def configure_sharded_model(self) -> None:
 class DataHooks:
     """Hooks to be used for data related stuff."""
 
+    __jit_unused_properties__ = ["prepare_data_per_node"]
+
+    def __init__(self) -> None:
+        super().__init__()
+        self._prepare_data_per_node: bool = True
```

Comment:
nit: this annotation is not necessary

```suggestion
        self._prepare_data_per_node = True
```

Reply:
@carmocca we also use pyre which prefers adding the explicit annotation here: https://github.com/facebook/pyre-check